docs/installation/kubernetes.md

```diff
@@ -0,0 +1,138 @@
+---
+id: kubernetes
+title: Kubernetes
+---
+
+## Kubernetes
+
+If you are using Kubernetes you may find the following scripts useful for a new installation of Pitchfork.
+
+They consist of a deployment, a service and a horizontal pod autoscaler that you can modify according to your needs.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pitchfork
+  labels:
+    app: pitchfork
+spec:
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  replicas: 1
+  selector:
+    matchLabels:
+      app: pitchfork
+  template:
+    metadata:
+      labels:
+        app: pitchfork
+        release: pitchfork
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8081"
+        prometheus.io/path: "/actuator/prometheus"
+    spec:
+      terminationGracePeriodSeconds: 0
+      containers:
+        - name: pitchfork
+          image: expediagroup/pitchfork:1.19
```

Comment:
```suggestion
          image: expediagroup/pitchfork:latest
```

Reply:
i'll add a comment at the top asking users to replace "latest" with the latest available version at the time.